### PR TITLE
add Task dev 7

### PR DESCRIPTION
**Pull request by kuznecowmihail**

### CODE REVIEW

File: Task_DEV-6/Task_DEV-6/Auto/CarGetter.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        public IEnumerable<Auto> Cars { get; }
+        static CarGetter carGetter;
+
+        /// <summary>
+        /// Private constructor of AutoGetter.
+        /// </summary>
+        /// <param name="nameXML">Name XML</param>
+        private CarGetter(string nameXML)
+        {
+            NameXML = nameXML;
```
Comment (AlexeiRomankov) [inline Task_DEV-6/Task_DEV-6/Auto/CarGetter.cs]: please use 'this'

File: Task_DEV-6/Task_DEV-6/Auto/CarGetter.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+        /// <summary>
+        /// Method returns list of auto.
+        /// </summary>
+        /// <returns autos>List of autos</returns>
+        public IEnumerable<Auto> GetAuto()
+        {
+            XDocument xDoc = new XDocument();
+            xDoc = XDocument.Load($"../../{NameXML}.xml");
+            return xDoc.Element("cars").Elements("car").Select(t => new Auto(
```
Comment (AlexeiRomankov) [inline Task_DEV-6/Task_DEV-6/Auto/CarGetter.cs]: too hard to read. it is better to split this logic

File: Task_DEV-6/Task_DEV-6/Auto/TruckGetter.cs [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+using System.Linq;
+using System.Xml.Linq;
+using System.Collections.Generic;
+
+namespace Task_DEV_6
+{
+    /// <summary>
+    /// Class singleton write cars into list from xml.
+    /// </summary>
+    public class TruckGetter
```
Comment (AlexeiRomankov) [inline Task_DEV-6/Task_DEV-6/Auto/TruckGetter.cs]: Is it enough differences between GetAuto for car and GetAuto for truck? If we add bus, electric car and etc. you will copy and paste this methods several times?

File: Task_DEV-6/Task_DEV-6/RequestHandler.cs [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+        /// </summary>
+        public void HandleRequest(Dictionary<string, ICommand> dictionaryOfCommands)
+        {
+            DictionaryOfCommands = dictionaryOfCommands ?? throw new NullReferenceException("Don't set a dictionary of command.");
+            // The existence of such a request.
+            bool existence = true;
+            // Request of users.
+
+            // Infinity cicle.
+            while (true)
```
Comment (AlexeiRomankov) [inline Task_DEV-6/Task_DEV-6/RequestHandler.cs]: it is better to iterate by variable

File: Task_DEV-6/Task_DEV-6/RequestHandler.cs [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+        /// </summary>
+        public RequestHandler()
+        {
+            this.CommandsForExecution = new List<ICommand>();
+        }
+
+        /// <summary>
+        /// Method handles request.
+        /// </summary>
+        public void HandleRequest(Dictionary<string, ICommand> dictionaryOfCommands)
```
Comment (AlexeiRomankov) [inline Task_DEV-6/Task_DEV-6/RequestHandler.cs]: lets try to rewrite without using continue

File: Task_DEV-6/Task_DEV-6/Auto/TruckGetter.cs (hunk 3 above)
Comment (kuznecowmihail) [reply Task_DEV-6/Task_DEV-6/Auto/TruckGetter.cs]: My teacher spoke that for this example use different class

**Pull request closed without merging**